configurations/create_dadaab_wusc_pool.py

```diff
@@ -0,0 +1,114 @@
+from core_data_modules.cleaners import swahili
```

Comment:
suggest renaming to something like create_wusc_pool_dadaab_messages since it's not specifically creating a dadaab-wusc-pool, but rather just the dadaab data to the wusc pool.